modules/sdk-core/src/bitgo/tss/eddsa/utils.ts

```diff
@@ -0,0 +1,20 @@
+import crypto from 'crypto';
+
+/**
+ * @param seed - used to construct derivation path deterministically
+ * @param isMaster - if set, path starts with prefix `m/`
+ * @return path `(m/)/999999/a/b` where `a` and `b` are 7-byte pseudorandom numbers based on seed
+ */
+export function getDerivationPath(seed: string, isMaster = true): string {
```

Comment:
Nothing here seems eddsa specific, can this be a generic util instead? Or conversely can we verify that the logic is the same or different with ecdsa?

Reply:
Just want to avoid devs re-creating this function and introducing differences in logic.